src/bridge/transactions/peg_in_confirm.rs

```diff
@@ -128,6 +141,15 @@ impl PegInConfirmTransaction {
         push_taproot_leaf_unlock_data_to_witness(&mut self.tx, input_index, unlock_data);
```

Comment:
We could reuse `push_depositor_signature_input_0()` function here.

Just call `self.push_depositor_signature_input_0(schnorr_signature);`

Reply:
Fixed as suggested